utils/seed.js

```diff
@@ -0,0 +1,93 @@
+const controllers = require('../controllers');
```

Comment:
Tell ESLint to shut up
```suggestion
/* eslint-disable no-restricted-syntax */
/* eslint-disable no-console */
const controllers = require('../controllers');
```